doc/api/tls.markdown

```diff
@@ -50,29 +50,41 @@ openssl pkcs12 -export -in agent5-cert.pem -inkey agent5-key.pem \
   - `certfile`: all CA certs concatenated in one file like
     `cat ca1-cert.pem ca2-cert.pem > ca-cert.pem`
 
+## ALPN, NPN and SNI
```

Comment:
This should not have made it into the backport

ALPN support is only on v5.x +

https://github.com/nodejs/node/pull/2564

Please move this section